routing/services/simplesaml.yml

```diff
@@ -0,0 +1,16 @@
+services:
+    # default configuration for services in *this* file
+    _defaults:
+        public: false
+
+    SimpleSAML\Configuration:
+        factory: ['SimpleSAML\Configuration', 'getInstance']
+
+    SimpleSAML\Session:
+        factory: ['SimpleSAML\Session', 'getSessionFromRequest']
+
+    SimpleSAML\Auth\AuthenticationFactory:
+        class: SimpleSAML\Auth\AuthenticationFactory
```

Comment:
This seems to be used only in tests. Should we use it in the controllers as well?

Reply:
🤷‍♂ Your guess is as good as mine